### PR TITLE
README and updates for edl client package

### PR DESCRIPTION
**Pull request by laurenfrederick**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: packages/earthdata-login-client/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# @cumulus/launchpad-auth
```
Comment (chuckwondo) [inline packages/earthdata-login-client/README.md:1]: Is this heading correct?

File: packages/earthdata-login-client/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Utilities for OAuth authentication using [NASA Earthdata Login](https://urs.earthdata.nasa.gov/).
+
+## Versioning
+
+Cumulus uses a modified semantic versioning scheme and minor releases likely include breaking changes.
+
+Before upgrade, please read the Cumulus [release notes](https://github.com/nasa/cumulus/releases) before upgraded.
+
+It is strongly recommended you do not use `^` in your `package.json` to automatically update to new minor versions. Instead, pin the version or use `~` to automatically update to new patch versions.
```
Comment (chuckwondo) [inline packages/earthdata-login-client/README.md:11]: Didn't we recently agree to wrap at 80 columns for Markdown? Perhaps you weren't on the call or had dropped off? @yjpa7145 had brought this up.

File: packages/earthdata-login-client/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Usage
+
+```bash
+  npm install @cumulus/earthdata-login-client
```
Comment (chuckwondo) [inline packages/earthdata-login-client/README.md:16]: nit: no need for leading spaces

File: packages/earthdata-login-client/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## About Cumulus
+
+Cumulus is a cloud-based data ingest, archive, distribution and management prototype for NASA's future Earth science data streams.
+
+[Cumulus Documentation](https://nasa.github.io/cumulus)
+
+## Contributing
+
+To make a contribution, please [see our contributing guidelines](https://github.com/nasa/cumulus/blob/master/CONTRIBUTING.md).
```
Comment (chuckwondo) [inline packages/earthdata-login-client/README.md:27]: nit: I would remove "see our" from the link. Just link "contributing guidelines."

Comment (laurenfrederick) [reply packages/earthdata-login-client/README.md:29]: This is consistent with all the other docs so i'll leave it until they update them

**Pull request merged**